msal/build/strong_signing.md

```diff
@@ -0,0 +1,17 @@
+### Strong-naming general notes
+
+1. Strong naming is NOT a security feature
```

Comment:
We might want to add a link to the docs.ms article: https://docs.microsoft.com/en-us/dotnet/framework/app-domains/strong-named-assemblies

Reply:
good point

---
In reply to: [212448834](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1221#discussion_r212448834) [](ancestors = 212448834)